msbuild/Xamarin.Shared/Xamarin.Shared.targets

```diff
@@ -1782,17 +1782,40 @@ Copyright (C) 2018 Microsoft. All rights reserved.
 		</PackLibraryResources>
 	</Target>
 
-	<Target Name="_UnpackLibraryResources" Condition="'$(_CanOutputAppBundle)' == 'true'" DependsOnTargets="ResolveReferences;_CollectBundleResources">
+	<Target Name="_PrepareUnpackLibraryResources">
+		<PropertyGroup>
+			<_StampDirectory>$(IntermediateOutputPath)resourcestamps</_StampDirectory>
+		</PropertyGroup>
+		<ItemGroup>
+			<_UnpackLibraryResourceItems Include="@(ReferencePath);@(ReferenceDependencyPaths)">
+				<StampFile>%(FileName).stamp</StampFile>
+			</_UnpackLibraryResourceItems>
+		</ItemGroup>
+		<MakeDir SessionId="$(BuildSessionId)" Condition="'$(IsMacEnabled)' == 'true'" Directories="$(_StampDirectory)" />
```

Comment:
Should you move the `<MakeDir/>` down to `_UnpackLibraryResources`, just so it can skip on incremental builds?

This is probably would only be noticeable for remote iOS builds.

Reply:
done